### PR TITLE
calls changed to use service details

### PR DESCRIPTION
**Pull request by sockterrier**

### CODE REVIEW

**Comment by sockterrier** [comment]
the page is backed by the v2 api now for most things, and never loads all the complete services at once. it loads a single service if you change anything, or if you look at the config files.

File: Products/ZenUtils/controlplane/application.py [hunk 1]
```diff
@@ -418,6 +418,8 @@ class _DeployedAppConfigList(Sequence):
 
     def __init__(self, service, client):
         self._service = service
+        if not service._data.has_key("ConfigFiles"):
+            service._data["ConfigFiles"] = client.getService(service.id)._data["ConfigFiles"]
```
Comment (rDr4g0n) [inline Products/ZenUtils/controlplane/application.py]: theres an endpoint for grabbing service configs that will be faster `/api/v2/services/:serviceId/serviceconfigs`

File: Products/ZenUtils/controlplane/client.py [hunk 2]
```diff
@@ -141,14 +142,31 @@ def getChangesSince(self, age):
...
         body = ''.join(response.readlines())
         response.close()
         decoded = ServiceJsonDecoder().decode(body)
         if decoded is None:
             decoded = []
         return decoded
 
+    def updateServiceProperty(self, service, prop):
+        """
+        Updates the launch property of a service.
```
Comment (rDr4g0n) [inline Products/ZenUtils/controlplane/client.py]: Minor thing, if you make any other change, this Updates *any* property of a service, not just the launch property.

**Comment by rDr4g0n** [comment]
Neither issue i commented on is a big deal, so I went ahead and approved.

**Pull request merged**